recipes/cgal/all/conanfile.py

```diff
@@ -42,40 +32,28 @@ def _build_subfolder(self):
     def _configure_cmake(self):
         if not self._cmake:
             self._cmake = CMake(self)
-            self._cmake.definitions["WITH_CGAL_Core"] = self.options.with_cgal_core
-            self._cmake.definitions["WITH_CGAL_Qt5"] = self.options.with_cgal_qt5
-            self._cmake.definitions["WITH_CGAL_ImageIO"] = self.options.with_cgal_imageio
-            self._cmake.definitions["CGAL_HEADER_ONLY"] = self.options.header_only
+            self._cmake.definitions["CGAL_HEADER_ONLY"] = "TRUE"
             self._cmake.configure(build_folder=self._build_subfolder)
         return self._cmake
 
     def _patch_sources(self):
-        if tools.Version(self.version) < "5.3":
-            tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+        if scm.Version(self.version) < "5.3":
+            files.replace_in_file(self, os.path.join(self._source_subfolder, "CMakeLists.txt"),
                                 "CMAKE_SOURCE_DIR", "CMAKE_CURRENT_SOURCE_DIR")
```

Comment:
There no longer is an older version

(I can not make a suggestion since the diff is big)

Reply:
Yes, there are: in the config.yml:
```
  "5.0.4":
    folder: all
  "5.1.5":
    folder: all
  "5.2.4":
    folder: all
```

The change I made is that, even for those three older versions, now the only variant is the header-only one. That removes all the options.

Reply:
Thank for pointing it out, I must have had a filter on 🙈